packages/eslint-config-eventbrite-legacy/rules/style.js

```diff
@@ -64,10 +86,18 @@ module.exports = {
         // http://eslint.org/docs/rules/no-nested-ternary
         'no-nested-ternary': 'error',
 
+        // disallow trailing whitespace at the end of lines
+        // http://eslint.org/docs/rules/no-trailing-spaces
+        'no-trailing-spaces': 'error',
+
         // allow underscores in identifiers
         // http://eslint.org/docs/rules/no-underscore-dangle
         'no-underscore-dangle': 'off',
 
+        // disallow ternary operators when simpler alternatives exist
+        // http://eslint.org/docs/rules/no-unneeded-ternary
+        'no-unneeded-ternary': 'error',
```

Comment:
interesting! i wonder if we have any of these in our codebase ha i've certainly done `var ? var : other` before without thinking

Reply:
We've 100% done it. I'll have to fix the code. I may have to turn off the rule in core-js first and then fix the code in a separate diff if there are too many offenders